Tasks/VsTest/vstest.ts

```diff
@@ -49,6 +49,7 @@ try {
     var useNewCollectorFlag = tl.getVariable('tia.useNewCollector');
     var isPrFlow = tl.getVariable('tia.isPrFlow');
     var vsTestVersionForTIA: number[] = null;
+	var ignoreVstestFailure: string = tl.getVariable("vstest.ignoretestfailures");
```

Comment:
I think you mean stage1 and not mseng.  Touch base with Sanjay Malpani.  Lots of discussions around this.